### PR TITLE
Add Distribution Report in a new tab

### PR DESCRIPTION
**Pull request by abhishekshukla85**
distributionreport

### CODE REVIEW

**Comment by abhishekshukla85** [comment]
Thanks for the contribution @jmaxxz

File: templates/report.html [hunk 1]
```diff
@@ -394,13 +405,94 @@
...
+                  tickOptions: {
+                    formatString:'%.3f'
+                  }
+                },
+                yaxis: {
+                  label: 'Percent of responses',
+                  tickOptions: {
+                    formatString: function(val){
+                      console.log(val);
+                      return '%.1f%';}()
```
Comment (matteofigus) [inline templates/report.html]: Hi, thanks for your PR. 
I tried to execute this with some benchmarks locally, and I can see some `undefined` in my console, probably due to this console.log. Can you investigate? This is the file:
https://gist.github.com/matteofigus/61cb3ac662467b9f6076

Thanks,
Matteo

**Comment by abhishekshukla85** [comment]
@matteofigus I missed it. Fixed now.

**Comment by matteofigus** [comment]
Published in 0.2.4

**Pull request merged**